packages/test-utils/src/matchers/to-have-focus.spec.ts

```diff
@@ -12,6 +11,16 @@ declare global {
     }
 }
 
+expect.addSnapshotSerializer({
+    test(value: unknown) {
+        return typeof value === "string";
+    },
+    serialize(value: string) {
+        /* eslint-disable-next-line no-control-regex -- expected to replace control regex */
+        return value.replace(/\u001B\[[0-9;]*m/g, "");
```

Comment:
- `strip-ansi` drar in `ansi-regex` och kör `value.replace(ansiRegex, "")`.
- `ansi-regex` innehåller bara en mer komplicerad variant av detta regex (ok brutalt mycket mer komplext men inget vi behöver).